src/test/java/DukeTest.java

```diff
@@ -0,0 +1,50 @@
+import duke.*;
```

Comment:
Perhaps it would be better to list imported classes explicitly. 

1. This could keep the import list minimal and up to date. 
2. The class would also be easier to comprehend and maintain.

Reply:
Yes, I agree, I had missed this line when checking for code standard violations. Fixed it, thank you!